assets/app/scripts/directives/nav.js

```diff
@@ -32,7 +32,7 @@ angular.module('openshiftConsole')
             $('<option>')
               .attr("value", project.metadata.name)
               .attr("selected", project.metadata.name == $scope.projectName)
-              .text(project.displayName || project.metadata.name)
+              .text(project | displayName || project.metadata.name)
```

Comment:
@jwforres may know, but this looks to me like it should be pulling from annotation.